src/components/dashboard/GroupList.jsx

```diff
@@ -18,12 +18,18 @@ function GroupList({ groups, onGroupClick }) {
     </Avatar>
   );
 
-  const renderListItem = ({ title, imageURL, members }, index) => (
-    <List.Item onClick={() => onGroupClick(groups[index])} title={title}>
+  const renderTitle = (title, ind) => (
+    <div className={ind === index ? 'selected' : ''}>
+      {title}
```

Comment:
Noticed that the selected group is bold now 💯, but it's missing the ellipses styling when the group name is too long to be displayed.

Reply:
Good catch, I'll fix that.